docs/src/use/core-concepts/glossary.md

```diff
@@ -0,0 +1,397 @@
+
+Each node has a node type representing its type of syntax.
+For example, the `1 + 2` in the AST for `1 + 2;` is a _BinaryExpression_.
+
+ESLint uses the [ESQuery](#esquery) format for its AST and [selectors](#selector).
+
+## O
+
+### Override
+
+When a [config file](#config-file-configuration-file) sets a new severity and/or rule options for a rule in a set of files.
+
+Overrides are supported in both configuration formats.
+
+The following [flat config](#flat-config) overrides `no-unused-expression` from `"error"` to `"off"` in `*.test.js` files:
+
+```js
+export default [
+  rules: {
+    "no-unused-expressions": "off"
+  },
+  {
+    files: ["*.test.js"],
+    rules: {
+        "no-unused-expressions": "off"
+    }
+  }
+];
+```
+
+...and the following [legacy config](#flat-config) also overrides `no-unused-expression` from `"error"` to `"off"` in `*.test.js` files:
+
+```js
+{
+  "rules": {
+    "no-unused-expressions": "off"
+  },
+  "overrides": [
+    {
+      "files": ["*.test.js"],
+      "rules": {
+        "no-unused-expressions": "off"
+      }
+    }
+  ]
+}
+```
+
+Overrides are much simpler in flat configs, as flat configs declare all rules in one file rather than allowing nested hierarchies of configs the way legacy configs do.
+
+For more information on overrides in legacy configs, see [Configuration Files (Deprecated) > How to overrides work?](../configure/configuration-files-deprecated#how-do-overrides-work).
+
+## P
+
+### Parser
+
+A package that reads in a string and converts it to an [AST](#abstract-syntax-tree-ast).
+
+By default, ESLint uses [Espree](https://github.com/eslint/espree) parser, which is compatible with standard JavaScript runtimes and versions.
+
+Custom parsers let ESLint parse non-standard JavaScript syntax.
+Often custom parsers are included as part of shareable configurations or plugins, so you don’t have to use them directly.
+For example, [@typescript-eslint/parser](https://www.npmjs.com/package/@typescript-eslint/parser) is a custom parser included in the [typescript-eslint](https://typescript-eslint.io) project that lets ESLint parse TypeScript code.
+
+For more information on using parsers with ESLint, see [Configure a Parser](../configure/parser).
+
+### Plugin
+
+A package that can contain a set of [configurations](#shareable-config-configuration), [environments](#environment), [processors](#processor), and/or [rules](#rule).
+
+A popular use case for plugins is to enforce best practices for a framework.
+For example, [@angular-eslint/eslint-plugin](https://www.npmjs.com/package/@angular-eslint/eslint-plugin) contains best practices for using the Angular framework.
+
+For more information, refer to [Configure Plugins](../configure/plugins).
+
+### Processor
+
+A part of a plugin that extracts JavaScript code from other kinds of files, then lets ESLint lint the JavaScript code.
+Alternatively, processors can convert JavaScript code during preprocessing.
```

Comment:
This isn't very clear. Convert JavaScript into what?

Reply:
Just removed, I don't think what I was going for here (being thorough in what they can do) actually adds any new info.